region-details.html

```diff
@@ -7,11 +7,7 @@ <h1>Browse by Region - <span data-template="app:parameter" data-template-name="r
       <p>Listed below are all the news reports on Wilde’s trials that were published in <span
           data-template="app:parameter" data-template-name="region"/>. Select one of the reports to
         navigate directly to it.</p>
-    </div>
-  </div>
-  <div class="row">
-    <div class='col-md-6'>
-      <p>Listed below are all the news reports on Wilde’s trials that were published in one specific region. Select one of the reports to navigate directly to it.</p>
+      
       <div data-template="app:details-region"/>
     </div>
   </div>
```

Comment:
This change would also put everything into the page header. You could follow the example above to put the paragraph below the horizontal line.